docs/Tumblr-Classes.md

```diff
@@ -11,21 +11,27 @@ These are the stylesheet classes used by Tumblr on their pages.
 
 ### Data attributes
 
-* `data-post-id` for post ID
-* `data-type` for post Type
+* `data-post-id` for post ID of the post
+* `data-type` for post Type of the post
 * `data-following-tumblelog` if following that user
 * `data-tumblelog-name` for owner of the post
+* `data-root-id` for the post ID of the original post
+* `data-json` the contents of the post in json format
+* `data-is-animated` whether the original photoset or video has animation
 
 ### Post Types
 
+* .is_regular
```

Comment:
I'm not sure regular and reblog are posts types in the same way that the rest of these are... They should probably go in "others" instead.

They definitely need more explanation if we're going to document them usefully.